CatchIoScriptImpl/Items/ThrowableItem.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace CatchIoScriptImpl.Items
+{
+    abstract class ThrowableItem : Item
+    {
+        public float CalculateProjectileMovePoints(float startPos, float endPos)
```

Comment:
I think this method should be delete because projectile arc movepoints will be calculate on TopDownPhysics2D class.

Reply:
You're right. I'll remove that